lib/Models/CustomComponents.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+import React from 'react';
```

Comment:
There might be a bigger issue here, I've just realised - `Models/registerCustomComponents` also requires `React`.
Would you agree that one ought to go in a new `ReactCustom` directory (or similar)?